src/utils/event-manager.ts

```diff
@@ -22,29 +66,101 @@ export class EventManager extends Construct {
    * The event bus to register new rules with.
    * @attribute
    */
-  private readonly eventBus: IEventBus;
+  public readonly eventBus: IEventBus;
+
+  public readonly applicationPlaneEventSource: string = 'testApplicationPlaneEventSource';
+  public readonly controlPlaneEventSource: string = 'testControlPlaneEventSource';
+  // sensible defaults so they are not required when instantiating control plane
+  readonly eventMetadata: EventMetadata = {
```

Comment:
See [TS Playground](https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBAYwDYEMDOa4CUCuThwDeAUHGXCgCaUAqKUA5sDABQz1MwBcFAdgJ4BKHigFEAvsXEBuYqEiw4AS14xgUAGYoEBAJK58dRswmz50eCrWbtBAKIA3YKoAKKGNd5FS5NBBxQOjxoMFAqDADaALoAND5klMwoSkg0-GDAwaHh0aSScuAWcDDp9k6qALLMlO4ocAC83mQRANbA-Mpejs4waRlRAPw83a7unrL55orOOAC2cCO9pU1wAPIAcgBCqwCCWAAiuusA4gD6WHYAigCqdgDKNA1wAOQQvABGEPSU4VjAAI44YAhZ5xMgbbZ7Q4nU53a4AYXh9zuT1eHy+UB+vAYdxwCB0GFBPghuwORzOADEdroADLXC6ot6fb7hCnJJABYBE8EUimQskwi43e6PRqvDQaZmY34AoEgsFrXn86FnOGI5GMiVSrE4vEEtDcxV80kq05U2n0uyayUYnVslKcw0uLCrABqujuug2sIRSLuKLFYCgEAcSjQSjeuPxwINCudbo9XvWZupdIZgeDofDb3tHKgXIV+zs8fdnu9ar9AZeiSDIbDEd4Uf1hqLJcT3vNaatYprmfrOfZjoVm1pNPJPvV-tR7xSSHCTZjhpHNLHMM7luns9Zg-zhp28JouldOxodnOV1uD1R2hgSgc7mAf0BwJge4PR5PZ4rGrFN7vD4XQkFX3Q9j1PFMLXTF4-3vNRcyHHwixAj9wKFS9RWrYAYIfJ85VfQs7GQsCv19H9MOwtRANjRDCPfYiIK7VFEgo4B4N3OJJkKRRkHQTBFgqUQUCYKAVjIIN-zUOB8yoN4kA6KA8GBHhCDgVp2iyMJsSiHgDAIcQnkIGR4ikrDKFkjo0BwMAFDUShFjQYZyhgKoYBqdhoieCJjPIIg4CZW0ZWfEJlLgPwAiCVTngQN5QggJAXFQXhgDufxAi5WI4ESdgUj6TJVMWXKADoSShcc0JFKJ9PEBUfLIFT-JZbE2LylSwrSngIiimLg3ixLktSnRngyrL2VyjqCtKYqthNcd1wuSq4GquBvPIer0Ua3VowwEK2oizrotUHqEtEfrwvSmJMqSHLSnGpyipKgVVVI-0FuqlayAAeg+hYYAQQrjKiWRjPE2CCAOpKEFs3SHLgASwAAHgmjILt0gA+J4koAd1hlAEaR4AUcU1GWEEIGfJBh84E4VYoHhaS1F0lhgDum6FhZjJhGwRTRNq-B4AU-AnhgAALMNCvB4BIeAShocKzgmfZ4BSfe5QNDgFgAEIBaVnnavIbWMeAbHGY+gAqOB3lQYWLatuBTY+5W9bISQnZMmAAi8bXZB8-JyZwS2lAQChqCMTgaAgRYFZ6ATeCE9RFkcnpcou9hjG4OB9EU0PmE5hwICUShdfIEWxep2n6cfRSo8qQThMWQRCqoWgOGYNgW5gR3yHycQgA) for suggestion. Happy to chat about this.

Reply:
updated EventManager